ui/component/app/view.jsx

```diff
@@ -160,6 +160,17 @@ function App(props: Props) {
     return () => window.removeEventListener('mouseup', handleForwardAndBackButtons);
   });
 
+  // allows user to pause miniplayer using the spacebar without the page scrolling down
+  useEffect(() => {
+    const handleKeyPress = e => {
+      if (e.key === ' ' && e.target === document.body) {
+        e.preventDefault();
+      }
+    };
+    window.addEventListener('keydown', handleKeyPress);
+    return () => window.removeEventListener('keydown', handleKeyPress);
+  });
```

Comment:
The second argument to this effect should be `[]` so it isn't re-ran on every render

Reply:
Ahh good catch, thanks Sean! Updated